lib/bolt_ext/server.rb

```diff
@@ -28,8 +28,30 @@ class TransportAPI < Sinatra::Base
       if event[:type] == :node_result
         @r = event[:result].to_json
       end
+
+      [200, [@r]]
     end
+  end
+
+  post '/winrm/run_task' do
+    content_type :json
+
+    body = JSON.parse(request.body.read)
+    keys = %w[user password port connect-timeout ssl ssl-verify tmpdir cacert extensions]
+    opts = body['target'].select { |k, _| keys.include? k }
+    opts['protocol'] = 'winrm'
+    target = [Bolt::Target.new(body['target']['hostname'], opts)]
+    task = Bolt::Task.new(body['task'])
```

Comment:
In `examples/`, we pass `{ "task": { "name": "...", "file_content": "..." } }`. That doesn't seem to match the API we've documented. Do we still need to pass `name`? The example will also be wrong about where `file_content` goes.

Reply:
Nevermind, `name` is called out in the docs, I was overlooking the `file_content` restructuring. I think the examples are the only thing that's wrong. It's worth either updating or deleting them.

Reply:
Updated examples -  I had completely forgotten it was even there.